src/components/scenes/FioAddressSettingsScene.js

```diff
@@ -39,11 +39,37 @@ type Props = StateProps & DispatchProps & ThemeProps & OwnProps
 
 class FioAddressSettingsComponent extends React.Component<Props, LocalState> {
   state: LocalState = {
+    showAddBundles: false,
     showTransfer: false
   }
 
   componentDidMount(): * {
-    this.props.refreshAllFioAddresses()
+    const { refreshAllFioAddresses, route } = this.props
+    const { showAddBundles } = route.params
+    refreshAllFioAddresses()
+    if (showAddBundles) {
+      this.setState({ showAddBundles: true })
+    }
+  }
+
+  afterAddBundlesSuccess = (result: { bundles: number } | any) => {
+    const { refreshAllFioAddresses, navigation, route } = this.props
+    const { fioWallet, fioAddressName, refreshAfterAddBundles } = route.params
+
+    refreshAllFioAddresses()
+
+    this.setState({ showAddBundles: false })
+    showToast(s.strings.fio_request_add_bundles_ok_text)
```

Comment:
Please change all variables that have `bundles` to `bundledTxs` or `bundled_txs` bundles is not a proper name